components/VFBMain.js

```diff
@@ -956,7 +940,7 @@ export default class VFBMain extends React.Component {
       let _height = node.getRect().height;
       let _width = node.getRect().width;
       return (<div className="flexChildContainer" style={{ position : "fixed", overflow : "scroll", height: _height, width: _width }}>
-        <VFBGraph ref={ref => this.graphReference = ref} instance={this.instanceOnFocus} visible={graphVisibility} />
+        <VFBGraph instance={this.instanceOnFocus} dispatch={this.props.vfbGraph} visible={graphVisibility} />
```

Comment:
genuine question since I am bit lost, what is dispatch/this.props.vfbGraph intended to do? Thx

Reply:
The dispatch property wasn't used, has been removed during code cleaning.